### PR TITLE
Init french language

### PR DESCRIPTION
**Pull request by sepiropht**
[# issue 124](https://github.com/ferrous-systems/rust-three-days-course/issues/124)

### CODE REVIEW

File: presentation/chapters/fr-FR/overview.chapter [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+-   Adopté par Mozilla
+-   Présenté au grand public en 2012 en version 0.12
+-   Ressemblait un peu à Go à l'époque
+
+---
+
+## Focus
+
+-   Rust a perdu beaucoup de features entre 2012 et 2014
+-   Grabarge Collector, event runtime, sytème complexe gérer les erreurs
```
Comment (spacekookie) [inline presentation/chapters/fr-FR/overview.chapter]: Je ne connais pas beaucoup de francais, mais peut-etre que c'est un typo? :P

File: presentation/chapters/fr-FR/overview.chapter [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+-   Le design s'est orienté vers la programamtion système
+
+---
+
+## Développement
+
+-   Toujours avec un grand projet (ex: Servo)
+-   Une adoption très tôt des releases régulière, depecrations et RFC processus
+-   Bonnes methodes de developpement logiciel
+-   "Les choses ne sont pas complètes tant qu'il n'y a plus rien à ajouter mais quand il n'y a pas plus rien à enlever."
```
Comment (spacekookie) [inline presentation/chapters/fr-FR/overview.chapter]: I would break this line to be shorter

File: presentation/chapters/fr-FR/overview.chapter [hunk 3]
```diff
@@ -0,0 +1,139 @@
...
+-   Pragmatism et intégration facile
+-   Projet approchable
+
+---
+
+## Rust veut être en premier utilisable!
+
+---
+
+Plusieurs exemple dans ce cours sont vraiment court, c'est pourquoi nous allons passer du temps pour discuter de l'impact plusieurs features sur des larges projets.
```
Comment (spacekookie) [inline presentation/chapters/fr-FR/overview.chapter]: Also break this sentence

File: presentation/chapters/fr-FR/overview.chapter [hunk 4]
```diff
@@ -0,0 +1,139 @@
...
+-   Rust aide pour beacoup de chose difficile et vous fait confiance pour le reste
+
+---
+
+## Les features du languages
+
+
+-   Rust est un language basé sur les expressions. Chaques expressions a une valeur, qui peut être assignée ou retournée.
+
+-   "Paye ce que tu utilise": Rust a des features qui ont un coût au runtime explicitement. Les features non utilisés ne coûte rien.
```
Comment (spacekookie) [inline presentation/chapters/fr-FR/overview.chapter]: Also break these sentences. Should fit into ~ 100 characters to be nicely readable without having to soft wrap

File: presentation/chapters/fr-FR/overview.chapter [hunk 5]
```diff
@@ -0,0 +1,141 @@
...
+-   Bon support des FFI
+-   Les messages d'erreurs du compilateur sont excellents.
+-   Rust aide pour beacoup de chose difficile et vous fait confiance pour le reste
+
+---
+
+## Les features du languages
+
+
+-   Rust est un language basé sur les expressions. Chaques expressions a une valeur, qui peut être assignée ou retournée.
```
Comment (spacekookie) [inline presentation/chapters/fr-FR/overview.chapter]: Another line break here please

**Comment by sepiropht** [comment]
Fixed !

**Pull request merged**